default.nix

```diff
@@ -3,10 +3,11 @@ let
   sbt-derivation = import sources.sbt-derivation;
   pkgs = import sources.nixpkgs { overlays = [ sbt-derivation ]; };
   inherit (import sources."gitignore.nix" { inherit (pkgs) lib; }) gitignoreSource;
+  version = builtins.elemAt (builtins.match "^.*[\"](.+)[\"].*$" (builtins.readFile ./version.sbt)) 0;
```

Comment:
I don't recommend defining to `let` values until it needs to be refactored. From experience, Nix files can get complicated really fast so KISS principle holds well :)